storage/kv.go

```diff
@@ -79,6 +79,9 @@ type KV interface {
 	// Restore restores the KV store from a backend.
 	Restore(b backend.Backend) error
 	Close() error
+
+	// GetAuthStore returns the auth store that consists auth related metadata.
+	GetAuthStore() AuthStore
```

Comment:
remove this. kv is only for kv api. we should have a separate pkg for authstore as i mentioned above.